skypy/galaxies/velocity_dispersion.py

```diff
@@ -0,0 +1,57 @@
+r"""Models of galaxy velocity dispersion.
+
+"""
+
+import numpy as np
+from skypy.utils.random import schechter
+
+__all__ = [
+    'schechter_vdf',
+]
+
+def schechter_vdf(alpha, beta, vd_star, vd_min, vd_max, size=None, resolution=1000):
+    r"""Sample velocity dispersion of elliptical galaxies in the local universe following a Schecter function.
+
+    Parameters
+    ----------
+    vd_min, vd_max: int
+        Lower and upper bounds of random variable x. Samples are drawn uniformly from bounds.
+    resolution: int
+        Resolution of the inverse transform sampling spline. Default is 100.
+    size: int
+        Number of samples returned. Default is 1.
+
+    Returns
+    -------
+    velocity_dispersion: array_like
+        Velocity dispersion drawn from Schechter function.
+
+    Notes
+    -----
+    The probability distribution function :math:`p(\sigma)` for velocity dispersion :math:`\sigma`
+    can be described by a Schechter function (see eq. (4) in [1]_)
+
+    .. math::
+
+        \phi = \phi_* \left(\frac{\sigma}{\sigma_*}\right)^\alpha
+        \exp\left[-\left( \frac{\sigma}{\sigma_*} \right)^\beta\right] 
+        \frac{\beta}{\Gamma(\alpha/\beta)} \frac{1}{\sigma} \mathrm{d}\sigma
+
+        where :math:`\Gamma` is the gamma function, :math:`\sigma_*` is the charactersitic velocity dispersion, :math:`\phi_*` 
```

Comment:
```suggestion
      where :math:`\Gamma` is the gamma function, :math:`\sigma_*` is the characteristic velocity dispersion, :math:`\phi_*` 
```
I think the wrong rendering comes from the wrong indentation. Please verify that this aligned with the other text.